extensions/browser-ai/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "browser-ai",
-  "title": "Browser AI",
-  "description": "Use Perplexity models and Raycast Browser Extension to extend your browsing.",
-  "icon": "perplexity.png",
+  "title": "OpenRouter - Browser AI",
```

Comment:
Isn't the name here a bit misleading if it still works with perplexity?

Reply:
Yeah, changed back to "Browser AI"